src/utils/yarn.js

```diff
@@ -122,3 +122,11 @@ export async function bin(cwd: string) {
     tty: true
   });
 }
+
+export async function info(cwd: string, dependencies: Array<string> = []) {
```

Comment:
`dependencies`?

The arguments to `yarn info` are `yarn info [pkg] <key>`:

```js
$ yarn info lodash
{ name: 'lodash',
  description: 'Lodash modular utilities.',
  'dist-tags': { latest: '4.17.4' },
  versions:
   [ '0.1.0',
     '0.2.0',
     '0.2.1',
     '0.2.2',
     '0.3.0',
     '0.3.1',
     '0.3.2',
     '0.4.0',
     '0.4.1'
     ... }
$ yarn info lodash versions
[ '0.1.0',
  '0.2.0',
  '0.2.1',
  '0.2.2',
  '0.3.0',
  '0.3.1',
  '0.3.2',
  '0.4.0',
  '0.4.1',
  ... ]
```

We should make the signature `info(cwd: string, pkg: string, key?: string)`

Reply:
We've to send an array of arguments to the spawn process (https://github.com/boltpkg/bolt/blob/master/src/utils/processes.js#L67). Renamed to `dependencies` to `spawnArgs`.